package.json

```
@@ -12,7 +12,8 @@
     "dist:transpile:cjs": "rm -rf lib && cross-env BABEL_MODULE=commonjs babel src --out-dir lib --extensions '.js,.ts,.jsx,.tsx'",
     "dist:transpile:esm": "rm -rf lib-esm && babel src --out-dir lib-esm --extensions '.js,.ts,.jsx,.tsx'",
     "dist:types": "rm -rf types && tsc",
-    "lint": "eslint src docs/components",
+    "lint": "eslint '**/*.{js,ts,tsx}'",
+    "lint-fix": "eslint '**/*.{js,ts,tsx}' --fix",
```

[nit] I'm not sure if we have a naming convention for npm scripts, but `lint:fix` seems to be more consistent with the other scripts than `lint-fix`.

```suggestion
    "lint:fix": "eslint '**/*.{js,ts,tsx}' --fix",
```

---

Great call, I should have looked at the others to see what the normal pattern was so far